editor/debugger/editor_debugger_inspector.cpp

```diff
@@ -55,9 +55,14 @@ bool EditorDebuggerRemoteObject::_get(const StringName &p_name, Variant &r_ret)
 }
 
 void EditorDebuggerRemoteObject::_get_property_list(List<PropertyInfo> *p_list) const {
-	p_list->clear(); //sorry, no want category
-	for (const PropertyInfo &E : prop_list) {
-		p_list->push_back(E);
+	p_list->clear(); // Sorry, no want category.
```

Comment:
Not sure what this comment means, beside the bad grammar :)

Reply:
It seems to be clearing the list to get rid of any PropertyInfo with `PROPERTY_USAGE_GROUP` so the properties that get added to the list are not grouped in a category.